pkg/agent/manager/cache/cache.go

```diff
@@ -445,6 +527,99 @@ func (c *Cache) GetStaleEntries() []*StaleEntry {
 	return staleEntries
 }
 
+// SyncSVIDsWithSubscribers will sync svid cache:
+// entries with active subscribers which are not cached will be put in staleEntries map
+// records which are not cached for remainder of max cache size will also be put in staleEntries map
+func (c *Cache) SyncSVIDsWithSubscribers() {
+	c.mu.Lock()
+	defer c.mu.Unlock()
+
+	c.syncSVIDsWithSubscribers()
+}
+
+// Notify subscribers of selector set only if all SVIDs for corresponding selector set are cached
+// It returns whether all SVIDs are cached or not.
+// This method should be retried with backoff to avoid lock contention.
+func (c *Cache) Notify(selectors []*common.Selector) bool {
```

Comment:
I _think_ the current cache behavior is such that we'll return a partial list.... should this function return true if at least one SVID is available?

Reply:
Yep that's the intentional change..we want to return complete list of identities to users mainly for users would just assume first result has all identities it needs(e.g cli users).